azure-function/dotnet/Models/Vehicle.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace GetBusData.Models;
+
+public sealed class Vehicle
+{
+    [JsonProperty("trip")]
```

Comment:
I don't think this `JsonProperty(...)` is necessary because the function app applies camelCasing by default.